lib/rules/object-shorthand.js

```diff
@@ -67,8 +87,52 @@ module.exports = function(context) {
 
 };
 
-module.exports.schema = [
-    {
-        "enum": ["always", "methods", "properties", "never"]
-    }
-];
+module.exports.schema = {
+    "anyOf": [
+        {
+            "type": "array",
+            "items": [
+                {
+                    "enum": ["always", "methods", "properties", "never"]
+                }
+            ],
+            "minItems": 0,
+            "maxItems": 1
+        },
+        {
```

Comment:
I think we can omit this second item to ensure people always specific the first option when specifying the second.